scipy/optimize/tests/test_linprog.py

```diff
@@ -2199,6 +2199,9 @@ class TestLinprogHiGHSMIP():
     method = "highs"
     options = {}
 
+    @pytest.mark.xfail(condition=sys.maxsize < 2 ** 32,
```

Comment:
An argument could be made to scope this to 32-bit Linux since I haven't seen the segfault on 32-bit Windows so far.

Reply:
It's also a GCC toolchain though, so it's fine to disable completely on 32-bit I'd say. Especially given that the upstream issue was already closed as "32-bit, so low prio"